application/squeeze2cast/squeeze2cast.c

```diff
@@ -830,38 +848,42 @@ static bool mDNSsearchCallback(mdnssd_service_t *slist, void *cookie, bool *stop
 	// walk through the list for device that expire on timeout
 	for (int i = 0; i < MAX_RENDERERS; i++) {
 		Device = glMRDevices + i;
-		if (!Device->Running || Device->Config.RemoveTimeout < 0 || !Device->Expired ||
-			now < Device->Expired + Device->Config.RemoveTimeout*1000 ||
-			(!Device->Config.RemoveTimeout && CastIsConnected(Device->CastCtx)))
+		if (!Device->Running || Device->Config.RemoveTimeout < 0 // unused entry, or device which never expires
```

Comment:
I would rather have all the continue under a single if stance. Is there a reason for not doing so?

Reply:
No need to separate them. I will make it a single if statement then if you prefer.